samples/zigbee/ncp/src/main.c

```diff
@@ -102,7 +102,11 @@ static void perform_custom_indication(zb_uint8_t led_idx)
 	zb_buf_get_out_delayed_ext(custom_indication, led_idx, 0);
 }
 
+#if (defined ZBOSS_PLATFORM_MAJOR) && (ZBOSS_PLATFORM_MAJOR < 5U)
```

Comment:
Why is this check introduced? Do we expect people using this sample with older Zboss? If so, it does not defend against using it with older Zboss which lacked vendor specific commands feature.

Reply:
That's correct.